docs/reference/packages.rst

```diff
@@ -38,24 +38,28 @@ install third-party libraries. The simplest way is to download a file directly::
 When installing a file directly, the ``target`` argument is still supported to set
 the destination path, but ``mpy`` and ``version`` are ignored.
 
-The URL can also start with ``github:`` as a simple way of pointing to content
-hosted on GitHub::
+The URL can also start with ``github:`` or ``gitlab:`` as a simple way of pointing to content
+hosted on GitHub: or GitLab::
 
     >>> mip.install("github:org/repo/path/foo.py")  # Uses default branch
     >>> mip.install("github:org/repo/path/foo.py", version="branch-or-tag")  # Optionally specify the branch or tag
+    >>> mip.install("gitlab:org/repo/path/foo.py")  # Uses default branch
+    >>> mip.install("gitlab:org/repo/path/foo.py", version="branch-or-tag")  # Optionally specify the branch or tag
 
 More sophisticated packages (i.e. with more than one file, or with dependencies)
 can be downloaded by specifying the path to their ``package.json``.
 
     >>> mip.install("http://example.com/x/package.json")
     >>> mip.install("github:org/user/path/package.json")
+    >>> mip.install("gitlab:org/user/path/package.json")
```

Comment:
Just wondering but do we really want to repeat this in every single example in all involved readme/.rst files? Because suppose we add another method then we'd also have to adjust all locations again?

Reply:
I just consider that example should be explicit to ease search and understanding.
I've done it searching for _github:_, and now _gitlab:_.
```shell
$ grep -rl "github:\|gitlab:"
tools/mpremote/README.md
tools/mpremote/mpremote/mip.py
tools/mpremote/mpremote/main.py
docs/reference/packages.rst
docs/reference/mpremote.rst
```